bliss/surveys/sdss.py

```diff
@@ -2,88 +2,97 @@
 import gzip
 import warnings
 from pathlib import Path
+from typing import List, Tuple, TypedDict
 
 import numpy as np
-import pytorch_lightning as pl
 import torch
 from astropy.io import fits
 from astropy.wcs import WCS, FITSFixedWarning
 from einops import rearrange
 from scipy.interpolate import RegularGridInterpolator
 from torch import Tensor
-from torch.utils.data import DataLoader, Dataset
+from torch.utils.data import DataLoader
 
 from bliss.catalog import FullCatalog, SourceType
+from bliss.simulator.background import ImageBackground
+from bliss.simulator.prior import ImagePrior, PriorConfig
+from bliss.simulator.psf import ImagePointSpreadFunction, PSFConfig
+from bliss.surveys.survey import Survey
```

Comment:
Yep, make a lot sense. If a user wants to add new survey, they just import a few abstract classes and  inherit from them. Those UML diagrams paid off!